README.md

```diff
@@ -12,7 +12,7 @@ Containerization provides APIs to:
 - [Create an optimized Linux kernel for fast boot times](./kernel/).
 - [Spawn lightweight virtual machines and manage the runtime environment](./Sources/Containerization/LinuxContainer.swift).
 - [Spawn and interact with containerized processes](./Sources/Containerization/LinuxProcess.swift).
-- Use Rosetta 2 for executing x86_64 processes on Apple silicon.
+- Use Rosetta 2 for running containers using linux/amd64 images on Apple Silicon.
```

Comment:
Thanks for the PR!  
What do you think about wording to the effect of "Use Rosetta 2 for running linux/amd64 containers on Apple Silicon."